addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -784,3 +764,45 @@ a.no-decoration {
         border-radius: 50%;
     }
 }
+
+.o_wsale_share_btn {
+    &::after {
+        display: none;
+    }
+}
+
+.o_wsale_share_menu {
+    min-width: 3rem;
+}
+
+.o_total_border {
+    border: $border-width solid $border-color;
+    @include media-breakpoint-down(lg) {
+        border-width: 0;
+    }
+}
+
+.o_wsale_address_edit {
+    span[itemprop='name'] {
+        @extend h6;
```

Comment:
Definitely avoid that 😬
Never `@extend` utility-classes, these operations take a huge amount on compilation time. Can you apply `h6`  class directly? If it's not possible just do:
```scss
span[itemprop='name'] {
    font-size: $h6-font-size;
    font-weight: $headings-font-weight;
    line-height: $headings-line-height;
}
```